nav2_route/src/route_server.cpp

```diff
@@ -65,14 +65,20 @@ RouteServer::on_configure(const rclcpp_lifecycle::State & /*state*/)
   declare_parameter_if_not_declared(
     node, "max_planning_time", rclcpp::ParameterValue(2.0));
 
+  nav2_util::declare_parameter_if_not_declared(
```

Comment:
Move back into the GraphLoader object to declare and get the parameter value. I don't think we need it here in the main route server file. We can let that object grab it itself.

Reply:
(and remove the class member added to the server too)

Reply:
I think this also makes sense to have no default value, so if it throws and exception while getting it, it should be caught as a failure.